hashtable/hashtable.py

```diff
@@ -65,14 +100,27 @@ def get(self, key):
 
         Implement this.
         """
+        index =  self.hash_index(key)
+        cur = self.storage[index]
 
-    def resize(self):
+        while cur != None:
+            if cur.key == key:
+                return cur.value
+            cur = cur.next
+
+        return None
+
+    def resize(self, size):
         """
         Doubles the capacity of the hash table and
         rehash all key/value pairs.
-
         Implement this.
         """
+        increase = size - self.capacity
+        add_array = [None] * increase
+        self.storage = self.storage + add_array
```

Comment:
I believe that this will create a storage that is longer than intended. And it will also be a different length than the self.capacity will be set to

Reply:
you're absolutely right on this part. I mean, it passed the test, but I also realized that since the capacity changed, then the index would change, too. I just refactored this one and still trying to pass the test. Will let you know soon.